ContactDetailsApi/V2/Gateways/ContactDetailsDynamoDbGateway.cs

```diff
@@ -49,16 +54,41 @@ public async Task<List<ContactDetails>> GetContactDetailsByTargetId(ContactQuery
             return contactDetailsEntities.ToDomain();
         }
 
+        [LogCall]
+        public async Task<UpdateEntityResult<ContactDetailsEntity>> EditContactDetails(EditContactDetailsQuery query, EditContactDetailsRequest request, string requestBody)
+        {
+            _logger.LogDebug("Calling IDynamoDBContext.LoadAsync for {ContactId} {PersonId}", query.ContactDetailId, query.PersonId);
+
+            var existingContactDetails = await _dynamoDbContext.LoadAsync<ContactDetailsEntity>(query.PersonId, query.ContactDetailId).ConfigureAwait(false);
+            if (existingContactDetails == null) return null;
+
+            var updaterResponse = _updater.UpdateEntity(
+                existingContactDetails,
+                requestBody,
+                request.ToDatabase()
+            );
+
+            if (updaterResponse.NewValues.Any())
+            {
+                _logger.LogDebug("Calling IDynamoDBContext.SaveAsync to update {ContactId} {PersonId}", query.ContactDetailId, query.PersonId);
```

Comment:
```suggestion
                _logger.LogDebug("Calling IDynamoDBContext.SaveAsync to update  with contactId:{ContactId} and personId:{PersonId}", query.ContactDetailId, query.PersonId);
```

Reply:
Not sure this is necessary with a templated log string, but I can add it